Ocelot.Sample.ApiGateway/Ocelot.json

```diff
@@ -0,0 +1,35 @@
+{
+  "Routes": [
+    {
+      "DownstreamHostAndPorts": [
```

Comment:
Why DownstreamHostAndPorts? You are using consul, you don't need that

Reply:
How to register my service which are defined are defined in different appsettings.json?

I am getting error in program.cs

Here is my another branch on which I am trying to understand your suggestions.

@ggnaegi I need help here

https://github.com/ks1990cn/Ocelot/blob/74fc6a9fb3d88e06dc4001b5c45e8e6961e0713d/Ocelot.Downstream.Service.Instance1/Program.cs#L11-L25
![image](https://github.com/ThreeMammals/Ocelot/assets/29522704/7385021d-bfae-45d4-b708-04774ec9141e)